projects/rbenv.org/ruby-build/package.yml

```diff
@@ -0,0 +1,37 @@
+distributable:
+  url: git+https://github.com/rbenv/ruby-build.git
+  ref: ${{version.tag}}
+
+versions:
+  github: rbenv/ruby-build
+
+dependencies:
+  gnu.org/autoconf: ^2.72
```

Comment:
is this really needed?

Reply:
Most of these deps are `companions` at most and not `dependencies`. Since without them the binaries will run they just error for specific sub commands.

As a rule of thumb, if `--help` runs without the dep then its a companion.